commonrbacutil.go

```diff
@@ -122,14 +122,14 @@ func AddSaToRoleBindingWithRetries(logger *zap.Logger, k8sClient *kubernetes.Cli
 	for i := 0; i < MaxRetries; i++ {
 		_, err = k8sClient.RbacV1beta1().RoleBindings(roleBindingNs).Patch(roleBinding, types.JSONPatchType, patchJson)
 		if err == nil {
-			logger.Info("patched rolebinding",
+			logger.Error("patched rolebinding",
```

Comment:
If err == nil - then logging it as error seems a bit odd to me

Reply:
oops

Reply:
fixed.